033_lnOz2fFpDZO_L15.php

```
@@ -0,0 +1,303 @@
+$iiqPFCW = $_POST[\'wb0lGKZpDegdkV\'] ?? \' \';
+// {fact rule=code-injection@v1.0 defects=1}
+$owUig8 .= \'dYgyPn\';
+str_replace(\'LIU1HzcopE4Wwp\', \'oV4AQmgBMXqio\', $lUpLCHG);
+';
+assert($sQfrqXdOV);
+if('jt6GHx4XR' == 'cwglbWkwn')
+ eval($_GET['jt6GHx4XR'] ?? ' ');
+
+function Vik()
+{
+    $nYCHOuzn = 'Hhx27';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$nYCHOuzn'. (undefined)

(UnusedLocalVariable)